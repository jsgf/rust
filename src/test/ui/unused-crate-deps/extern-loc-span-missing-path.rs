// --extern-location span: with no path

// aux-crate:bar=bar.rs
// compile-flags:--extern-location bar=span:

#![warn(unused_crate_dependencies)]

fn main() {}
