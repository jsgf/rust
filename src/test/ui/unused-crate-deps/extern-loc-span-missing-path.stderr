error: `--extern-location`: missing start offset on `span`

